tests/providers/test_adapter.py

```diff
@@ -26,6 +29,20 @@ def test_mappers(self):
             list(sorted(qiskit_gate_name_to_braket_gate_mapping.keys())),
         )
 
+    def test_convert_qiskit_to_braket_circuit(self):
+        """Tests convert_qiskit_to_braket_circuit works with parametric circuits."""
+
+        theta = Parameter("θ")
+        qiskit_circuit = QuantumCircuit(1, 1)
+        qiskit_circuit.rz(theta, 0)
+        braket_circuit = convert_qiskit_to_braket_circuit(qiskit_circuit)
+
+        braket_circuit_ans = Circuit().rz(  # pylint: disable=no-member
```

Comment:
Could we extend the test to include more complex gates like U3 and also mix parametric and non parametric gates?

Reply:
Yes, thank you!